packages/payment-processor/src/payment/erc20-escrow-payment_v1.ts

```diff
@@ -0,0 +1,391 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
The file name should (probably) be contract version agnostic

Reply:
ok.